python/cuml/test/test_fil.py

```diff
@@ -260,30 +262,33 @@ def test_fil_skl_classification(n_rows, n_columns, n_estimators, max_depth,
 
     if n_classes == 2:
         assert array_equal(fil_preds, skl_preds_int)
-        fil_proba = np.asarray(fm.predict_proba(X_validation))
-        fil_proba = np.reshape(fil_proba, np.shape(skl_proba))
-        assert np.allclose(fil_proba, skl_proba, 1e-3)
+    fil_proba = np.asarray(fm.predict_proba(X_validation))
+    fil_proba = np.reshape(fil_proba, np.shape(skl_proba))
+    proba_threshold = 1e-3 if n_classes == 2 else 0.1
```

Comment:
0.1 seems like a high threshold on probability, especially when 1e-3 is the threshold for 2 classes. Could you share some of the stats on what errors we're seeing in practice?

Reply:
Thank you! Turns out, it's consistently below 1e-4 with some margin. The worst errors turned out to be in binary classification. Below is the output of `np.histogram()` over the relative errors:

```
(array([360,  10,   2,   4,   0,   0,   0,   0,   0,   0,   0,   0]),
 array([0.0e+00, 1.0e-6, 3.0e-6, 9.0e-6, 2.7e-5,
       8.1e-5, 2.4e-4, 7.2e-4, 2.1e-3, 6.5e-3,  1.9e-2, 5.9e-2, 1.7e-1]))))
```